requirements/default.txt

```diff
@@ -1,8 +1,10 @@
 cell
 eventlet
 dnspython
-django
-django-celery
+django<1.6
+django-celery<3.1
```

Comment:
can we update this?

Reply:
Sure, but means compatibility work.

I pinned to old stuff to make the tests pass. Unpinned (new) versions broke it.

Reply:
I recommend merging as is, and opening another issue for updating to at least the latest released LTS versions available